arithmetic_operators/home_work/cost_in_rubles_and_kopecks.py

```diff
@@ -0,0 +1,17 @@
+from random import randint
+
+
+def find_cost_in_rubles_and_kopecks(cost: int) -> tuple[int, int]:
+    """
+       Data: cost of the product in kopecks
+       Task: find cost in rubles and kopecks
+    """
+    return cost // 100, cost % 100
+
+
+if __name__ == "__main__":
+    cost = randint(100, 999999)
```

Comment:
Well, it's not necessarily, but we can add details in the name 
```suggestion
    const_in_coins = randint(100, 999999)
```

Reply:
Changed too